src/infobar.as

```diff
@@ -11,6 +11,8 @@ namespace InfoBar {
 
         UI::SetNextWindowPos(int(Board::Position.x), int(Board::Position.y) + int(Board::BoardSize) + BoardMargin, UI::Cond::Always);
         UI::SetNextWindowSize(int(Board::BoardSize), 42, UI::Cond::Always);
+        auto team = Room.GetSelf().Team;
+        UI::PushStyleColor(UI::Col::WindowBg, UIColor::GetAlphaColor(UIColor::Brighten(team.Color, 0.75), 1));
```

Comment:
Coloring the infobar poses some contrast issues. During my test run, for example, I found it hard to look at the timer. I think a better approach would be to have something belonging on the infobar that tells you your team. It could very well be a colored text such as "Red Team".